src/main/java/seedu/diary/model/internship/InternshipApplication.java

```diff
@@ -2,13 +2,18 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.diary.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.diary.model.ListenerPropertyType.DISPLAYED_INTERNSHIP_DETAIL;
```

Comment:
Don't use the same property name for different purposes, it is an easy source of bugs.

Reply:
Just make a public static string in InternshipApplication and use that maybe

Reply:
Ah sorry, i was lazy, I will make a separate static variable for it